src/events/touch.js

```diff
@@ -13,8 +13,7 @@ import p5 from '../core/main';
  * unique touch as it moves. Each element in the array is an object with x, y,
  * and id properties.
  *
- * The touches[] array is not supported on Safari and IE on touch-based
- * desktops (laptops).
+ * The touches[] array is not supported in IE on touch-based devices.
```

Comment:
MDN says [desktop Safari still doesn't support `clientX/Y` in touch events](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events#browser_compatibility) and it's just mobile safari does, which is I think what the original description was intending to explain. Mind if we keep the old text for now?

Reply:
Sure but the way it's phrased makes it sound like it's talking about Safari in general and then IE on touch-based desktops and laptops. Both are pretty irrelevant points to make anyway since Macs don't have touch screens and IE was replaced by Edge years ago. I think the phrasing should be changed at least but yeah that's a separate issue I just didn't know how to make another fork so I added it to this repo.